test/config/karma.unit.js

```diff
@@ -90,10 +92,15 @@ module.exports = function(karma) {
   };
 
   if (collectTranslations) {
-    config.plugins = [].concat(config.plugins || [], 'karma-*', require('./translation-reporter'));
+    config.plugins = [].concat(config.plugins || ['karma-*'], require('./translation-reporter'));
```

Comment:
When having `config.plugins` defined, `karma-*` would not be set, ergo the test-suite not run. Or am I missing something?

Reply:
If defined, we need to include `karma-*`. This is the way karma works.

So if plug-ins are previously defined, they have to include `karma-*` already or nothing would work. If that makes sense :thinking:.

Reply:
Got it